### PR TITLE
Add MSP2_SENSOR_CONFIG_ACTIVE message

### PR DESCRIPTION
**Pull request by haslinghuis**
See: https://github.com/betaflight/betaflight/pull/13177

- This improves information as we can observe a sensor is included in cloud build, disabled or detected.
- Added separate `Sensor Info` panel.

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by nerdCopter** [comment]
`56f1f942` with #13177
image
https://build.betaflight.com/api/builds/88b72560782fed6f5dd754e7c0d60df2/json

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
